check/check_db_dump_recent.py

```diff
@@ -31,9 +34,8 @@ def run(self, log_file_path):
             # Use a inner try...except block to handle the more expected error
             # of "Couldn't open/read the provided file" differently.
             try:
-                with open(log_file_path, 'r') as log_file_path:
-                    lines = log_file_path.read().splitlines()
-                    last_line = lines[-1]
+                with open(log_file_path, 'r') as log_file:
+                    line_list = log_file.read().splitlines()
```

Comment:
Indirectly relevant to this PR, but does this log get rotated? Have you noted the performance of this script?

Reply:
The log does not currently get rotated. Extracting the whole file is not currently a performance issue.